mallard/client.py

```diff
@@ -146,7 +151,18 @@ def _clean(self, message) -> Optional[str]:
         )
 
         try:
-            result = await duckduckgo.get_zci(query)
+            with self.rl.try_run(message.guild.id) as ok:
+                if ok:
+                    # Ok to send query
+                    result = await duckduckgo.zci(query)
+                else:
+                    # This guild has hit the rate limit
+                    user = f"{message.author.name}#{message.author.discriminator}"
+                    logger.info(f"Rate limited! (guild: {message.guild.name}, user: {user})")
+                    rl_logger.info(f"Guild: {message.guild.id}, User: {message.author.id}")
```

Comment:
not csv? (just curious)

Reply:
Ah that's better. I was originally thinking JSON but I thought that would be a bit silly.